src/rabbit.erl

```diff
@@ -1236,6 +1271,112 @@ validate_msg_store_io_batch_size_and_credit_disc_bound(CreditDiscBound,
             end
     end.
 
+-spec product_name() -> string().
+
+product_name() ->
+    #{name := ProductName} = product_infos(),
+    ProductName.
+
+-spec product_version() -> string().
+
+product_version() ->
+    #{version := ProductVersion} = product_infos(),
+    ProductVersion.
+
+-spec product_infos() -> #{name := string(),
```

Comment:
Can we rename this to `product_info`?

💡 [Information is a mass noun and doesn't have a plural](https://www.grammar-monster.com/plurals/plural_of_information.htm) 💡

Reply:
`info` and `infos` (info items) is a term commonly used in the codebase, although in a different context (metric emission)

Reply:
I would have never guessed that `infos` stands for `info_items`.

`product_info_items` makes sense, even though I find `product_info` more succinct and easier to understand. Either of them work, while `product_infos` not so much.

Reply:
> bulb [Information is a mass noun and doesn't have a plural](https://www.grammar-monster.com/plurals/plural_of_information.htm) bulb

Thank you, I always forget about this rule...